package.json

```diff
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/liatrio/libotrio#readme",
   "dependencies": {
     "@slack/bolt": "^3.4.0",
+    "axios": "^0.21.4",
```

Comment:
is this only needed for the local health check?  if so, it could be a `devDependency` instead.

also, I'm partial to `node-fetch` myself, but I won't die on that hill 🙂

Reply:
The feature that the Panthers are working on will require an http client also.

The problem is that node-fetch is an ESM-only module so it doesn't work with `require`. Like you mentioned using `import` is probably better practice anyway so maybe this is another reason to switch over.

Reply:
Yeah, ESM is the future, so it would be better to switch sooner rather than later.  This is fine for now though.